editor/editor_properties_array_dict.cpp

```diff
@@ -517,6 +543,59 @@ void EditorPropertyArray::drop_data_fw(const Point2 &p_point, const Variant &p_d
 		emit_changed(get_edited_property(), array, "", false);
 		update_property();
 	}
+
+	if (drag_data.has("type") && String(drag_data["type"]) == "nodes") {
```

Comment:
Can be simplified to
```suggestion
	if (String(drag_data.get("type", "")) == "nodes") {
```